docs/source/glossary.md

```diff
@@ -47,4 +47,12 @@ Posterior
   Once you are satisfied with the model, posterior distribution can be summarized and interpreted. Common questions for the posterior include: intervals of defined boundaries, intervals of defined probability mass, and point estimates. When the posterior is very similar to the prior, the available data does not contain much information about a parameter of interest.
 
   - For more on generating and interpreting the posterior samples, see Statistical Rethinking 2nd Edition By Richard McElreath, chapter 3.
+
+Maximum a Posterior (MAP)
```

Comment:
We can use two titles in this case (sphinx lets you do that) , one for the proper name `Maximum a Posteriori` and one for the common abbreviation `MAP`

It would look like this:

```
Maximum a Posteriori
MAP
  Similar to MLE,...
```